inc/features/blocks/experience/register.php

```diff
@@ -0,0 +1,84 @@
+<?php
+/**
+ * Experience Block Server Side.
+ *
+ * @phpcs:ignoreFile HM.Files.FunctionFileName.WrongFile
+ */
+
+namespace Altis\Experiments\Features\Blocks\Experience;
+
+use const Altis\Experiments\ROOT_DIR;
+use function Altis\Experiments\Features\Blocks\get_block_settings;
+
+const BLOCK = 'experience';
```

Comment:
Just an idea on the labelling in the code here. I wonder if it would be simper / more descriptive if we actually called these "conditional-content" block, or something. Experience Block could still be in the UI labels if we wanted, but I'm wondering if "Experience Blocks" as our Altis marketing goes is about quite a lot more than just conditional content blocks. Given this is all a part of Experiments, is this the "Experience Block ™" or, is this the conditional-content _capability_ that we need to complete the larger XB narrative (which is about the full cycle from content creation, to measuring, insights, action, feedback to content loop).

Reply:
Yeah, my initial thought was that the `experience-block` component is about driving conditional content as you say, an AB test is a subset of conditional content so the underlying mechanism is pretty much the same. I was thinking why have some options at the experience block level to choose the type eg. "AB test" or "Personalised content". Might be overloading things a bit there and I haven't really though too hard about the kind of UI that would entail so two separate entry points might be best.

Reply:
Changed this to be an 'Experience block' category with a block called 'Personalised content' within it. We can add an experiment block later on.